ecommerce-product-page-main/styled.css

```diff
@@ -301,61 +265,63 @@ ul.list li a:hover{
 }
 
 
-
-
-
-
-
-
 .block2{
-
     flex: 1;
     margin-left: 60px;
-    
-    
+
 }
 
 
 .h5orange{
-
-        color: hsl(26,100%,55%);
-        width: 150px
+    color: hsl(26,100%,55%);
 }
 
 .h1font{
-
     font-family: 'Kumbh Sans', sans-serif;
     font-weight: 700;
-    width: 250px;
-    display: inline;
 }
 
-.pfont{
-    
+.description{
     font-family: 'Kumbh Sans', sans-serif;
     font-weight: 400;
     font-size: 13px;
     color: hsl(219, 9%, 45%);
-    width: 250px;
-    display: inline;
     
 }
 
-.containflex{
-
+.counterSection{
     display: flex;
-    
+    align-items: center;
+    gap: 12px;
+    margin-top: 20px;
 
 }
 
-.containflexA{
-
+.addCounter{
     display: flex;
-    
+    align-items: center;

```

Comment:
Why have two "align-content" css in the same container? as line 294 also has the "align-content" css

Reply:
align items and justify content are different. align items and align content are different. 

https://css-tricks.com/snippets/css/a-guide-to-flexbox/#aa-align-items